modules/docs/arrow-docs/docs/_data/sidebar.yml

```diff
@@ -0,0 +1,316 @@
+options:
```

Comment:
where are the apidocs?

Reply:
Yep, didn't actually update the menu items, my bad 🤦